TeachingRecordSystem/tests/TeachingRecordSystem.TestCommon/TestData.CreatePersonProfessionalStatusBuilder.cs

```diff
@@ -115,6 +113,37 @@ public CreatePersonProfessionalStatusBuilder WithInductionExemption(bool? isExem
             return this;
         }
 
+        internal async Task<Guid> ExecuteAsync(TestData testData)
+        {
+            var professionalStatus = new ProfessionalStatus()
+            {
+                PersonId = _personId!.Value,
+                QualificationId = QualificationId,
+                RouteToProfessionalStatusId = _routeToProfessionalStatusId!.Value,
+                Status = _status,
+                AwardedDate = _awardedDate,
+                TrainingStartDate = _trainingStartDate,
+                TrainingEndDate = _trainingEndDate,
+                TrainingSubjectIds = _trainingSubjectIds,
+                TrainingAgeSpecialismType = _trainingAgeSpecialismType,
+                TrainingAgeSpecialismRangeFrom = _trainingAgeSpecialismRangeFrom,
+                TrainingAgeSpecialismRangeTo = _trainingAgeSpecialismRangeTo,
+                TrainingCountryId = _trainingCountryId,
+                TrainingProviderId = _trainingProviderId,
+                ExemptFromInduction = _exemptFromInduction,
+                DegreeTypeId = _degreeTypeId,
+                CreatedOn = DateTime.UtcNow,
+                UpdatedOn = DateTime.UtcNow
+            };
+
+            await testData.WithDbContextAsync(async dbContext =>
+            {
+                await dbContext.ProfessionalStatuses.AddAsync(professionalStatus);
```

Comment:
```suggestion
               dbContext.ProfessionalStatuses.Add(professionalStatus);
```

Reply:
Does this work without `dbContext.SaveChanges()`?

Reply:
It always gets called from the person builder and all the changes are saved from there, but I've added SaveChanges.